### PR TITLE
Adding tests for the changes in swagger api spec for patch status in VMInstanceView, properties in AssessPatches API and patch settings in Windows and Linux Configuration in VMModel

### PR DESCRIPTION
**Pull request by rane-rajasi**
…VMInstanceView, properties in AssessPatches API and patch settings in Windows and Linux Configuration in VMModel

PR for related swagger changes: https://github.com/Azure/azure-rest-api-specs/pull/11768 and https://github.com/Azure/azure-rest-api-specs/pull/12295

### CODE REVIEW

**Comment by YalinLi0312** [comment]
@rane-rajasi Please make sure the related swagger PR gets merged, otherwise I can't start my review.

**Comment by rane-rajasi** [comment]
Related swagger PR is now merged

File: global.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "1.0.45"
   },
   "sdk": {
-    "version": "3.1.301",
+    "version": "5.0.100",
```
Comment (YalinLi0312) [inline global.json:6]: Why you jump the version from 3 to 5? Does "5.0.100" from the swagger updates?

**Comment by YalinLi0312** [comment]
@rane-rajasi I don't see updates in metadata, versions, and so on. Could you please follow the checklist below?
- [ ] Swagger review PR(s) link in PR description, and the PR(s) is merged
- [ ] Metadata file `eng/mgmt/mgmtmetadata_{RP}.txt` is updated
- [ ] Versions in `*.csproj` and `AssemblyInfo.cs` are updated, and follows semver
- [ ] Release notes in `*.csproj` is updated
- [ ] If there's a API-version update, the API tags in `Az.Sdk.props` are updated
- [ ] Generated code is not touched
- [ ] Proper test coverage

**Comment by YalinLi0312** [comment]
If no updates, will close it in next week. Thanks

**Comment by rane-rajasi** [comment]
Please follow up with Adam Sandor for updates regarding this PR. Since it is in compute-2020-12, Adam will have more details on the PR

**Comment by Sandido** [comment]
Closing for now.

File: global.json (hunk 1 above)
Comment (rane-rajasi) [reply global.json]: This is updated in the compute-2020-12 branch

**Comment by rane-rajasi** [comment]
@Sandido Updated the PR with latest from compute-2020-12. Please review

**Pull request closed without merging**